configuration/scripts/tests/baseline.script

```diff
@@ -69,21 +69,26 @@ if (${ICE_BASECOM} != ${ICE_SPVAL}) then
       echo "base_file: ${base_file}"
       echo "test_file: ${test_file}"
 
-      ${ICE_CASEDIR}/casescripts/comparelog.csh ${base_file} ${test_file}
-      set logstatus = $status
-
-      if ( ${logstatus} == 0 ) then
-        echo "PASS ${ICE_TESTNAME} complog ${ICE_BASECOM}" >> ${ICE_CASEDIR}/test_output
-        echo "Regression baseline and test dataset may be the same"
-      else if ( ${logstatus} == 1 ) then
-        echo "FAIL ${ICE_TESTNAME} complog ${ICE_BASECOM} different-data" >> ${ICE_CASEDIR}/test_output
-        echo "Regression baseline and test dataset are not the same"
-      else if ( ${logstatus} == 2 ) then
+      if ("${base_file}" == "" || "${test_file}" == "" ) then
```

Comment:
are you getting rid of comparelog.csh?

Reply:
It's still there, just below this line in the else and indented.  This if just provides a better error message if the either base or test file are not defined.  Up to now, it would produce a FAIL, now it produces a MISS which is more accurate.

Reply:
ok, I missed that part